pup.js

```diff
@@ -0,0 +1,24 @@
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+
+const url = 'http://localhost:3000'
+
+if (!url) {
+  throw "Please provide URL as a first argument";
```

Comment:
this url is now hardcoded, therefore this if is unused

Reply:
True!